charts/refinery/values.yaml

```diff
@@ -164,7 +164,7 @@ config:
     # Further, if the environment variable 'REFINERY_REDIS_HOST' is set it takes
     # precedence and this value is ignored.
     # Not eligible for live reload.
-    RedisHost: "refinery-redis:6379"
+    RedisHost: '{{ include "refinery.redis.fullname" . }}:6379'
```

Comment:
leaving this PR in draft until https://github.com/honeycombio/helm-charts/pull/194 is decided.